test/N3DataFactory-test.js

```diff
@@ -143,4 +143,31 @@ describe('DataFactory', function () {
       ));
     });
   });
+
+  describe('quadTerm', function () {
+    it('should return a quadTerm with the correct value', function () {
+      DataFactory.quadTerm(new Quad(
+        new NamedNode('http://ex.org/a'),
+        new NamedNode('http://ex.org/b'),
+        new Literal('abc'),
+        new DefaultGraph()
+      )).value
```

Comment:
@rubensworks We are violating object homomorphism here (`value` not a string); I'm afraid this will slow things down. We likely want to reconsider at the RDF/JS level.

Reply:
Good point, I didn't realise this.